src/Components/Shared/src/ElementReferenceJsonConverter.cs

```diff
@@ -8,7 +8,7 @@ namespace Microsoft.AspNetCore.Components;
 
 internal sealed class ElementReferenceJsonConverter : JsonConverter<ElementReference>
 {
-    private static readonly JsonEncodedText IdProperty = JsonEncodedText.Encode("__internalId");
+    private static readonly JsonEncodedText s_IdProperty = JsonEncodedText.Encode("__internalId");
```

Comment:
This should be `s_idProperty`. The character after the first underscore should be lower case.

Applies to all fields.